openapi_spec_validator/validators.py

```diff
@@ -338,10 +340,10 @@ def __init__(self, dereferencer):
 
     @wraps_errors
     def iter_errors(self, schema, value):
-        validator = OAS30Validator(
+        validator = OAS31Validator(
             schema,
             resolver=self.dereferencer.resolver_manager.resolver,
-            format_checker=oas30_format_checker,
+            format_checker=oas31_format_checker,
```

Comment:
Same here. Shouldn't this be dependent of which spec we use? Otherwise we will use OAS31Validator for all specs? 🤔